### PR TITLE
feat(chatbot): add chatbot

### PR DESCRIPTION
**Pull request by makidelille**
### Description of the Change

add chatbot

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
     "express": "~4.13.3",
     "lodash": "~3.0.1",
     "morgan": "~1.0.0",
+    "ovh-angular-chatbot": "git+https://github.com/ovh-ux/ovh-angular-chatbot.git",
```
Comment (antleblanc) [inline package.json]: Not yet released?

File: client/index.tpl.html [hunk 2]
```diff
@@ -58,6 +60,14 @@
         <script src="bower_components/cryptojs/lib/PBKDF2.js"></script>
         <script src="bower_components/cryptojs/lib/HMAC.js"></script>
         <script src="bower_components/cryptojs/lib/AES.js"></script>
+
+        <script src="bower_components/jquery-ui/jquery-ui.js"></script>
```
Comment (antleblanc) [inline client/index.tpl.html]: Does `wiredep` task gives you already `jquery-ui`?

Comment (makidelille) [inline client/index.tpl.html]: it apears it doesn't, have a look at the build `index.html`, the core of jquery-ui is missing :/

Comment (Jisay) [reply client/index.tpl.html]: strange. Take a look at https://github.com/ovh-ux/ovh-manager-telecom/blob/11534aaeb6efec93d4d18fcab1a96ae0c2be69ca/bower.json#L137. We override jquery-ui dependency with the plugins we need. 
Maybe you need another plugin to be loaded. If so feel free to modify bower.json :-)

Comment (makidelille) [reply client/index.tpl.html]: Thanks a lot! It's fixed now. couple of draggable dependencies were missing ;)

**Pull request merged**